### ISSUE TITLE
Replace RSS generator (node-rss) with dylang RSS generator (rss)

### ISSUE THREAD

**Issue by julianlam** [NodeBB/NodeBB#258]
We are currently using the package `node-rss` as our RSS generator, but this package relies on the libxml package to output XML, which requires a lengthy compile step that continually causes `npm` to exit with a non-zero status code. Because it actually _compiles_ the libxml library, it also means I cannot simply copy/paste the `node_modules` folder into different directories, as it the libxml library errors out.

Long story short, it's impossible to use, nobody upstream has any inclination to fix it, so I think we should switch to the `rss` package instead (more stars, more commits, more forks).
